docs/source/app-dev/custom-views/index.rst

```diff
@@ -0,0 +1,328 @@
+.. Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _custom-views:
+
+Custom Views
+############
+
+Custom views provides convenient features to continuously ingest data from a Ledger into a database,
```

Comment:
provide (singular)

Reply:
i just noticed you merged so .. if you want a full review let me know

Reply:
@katmurp thanks that would be great. I needed to have a first version in for a snapshot yesterday, happy to process any suggestions you have.

Reply:
no probs, give me a few hours please...

Reply:
I'm not seeing the suggestion button :( so I'll just add comments..